easybuild/easyconfigs/h/HH-suite/HH-suite-3.0-beta.3-intel-2018a.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'CMakeMake'
+
+name = "HH-suite"
+version = "3.0-beta.3"
+ffindex_commit = '74550c8bde3d5b450755ec4be5e9cd56f28a231b'
+
+homepage = 'https://github.com/soedinglab/hh-suite'
+description = """ The HH-suite is an open-source software package for sensitive protein sequence searching
+ based on the pairwise alignment of hidden Markov models (HMMs). """
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'https://github.com/soedinglab/hh-suite/archive/',
+    'https://github.com/soedinglab/ffindex_soedinglab/archive/',
+]
+sources = [
+    'v%(version)s.tar.gz',
+    '%s.tar.gz' % ffindex_commit,
```

Comment:
@smoors Please rename on download (I'd use a datestamp of that commit), cfr. http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources